projects/netgrif-components-core/src/lib/header/abstract-header-service.ts

```diff
@@ -38,6 +39,7 @@ export abstract class AbstractHeaderService implements OnDestroy {
     protected constructor(protected _headerType: HeaderType,
                           protected _preferences: UserPreferenceService,
                           @Optional() private _viewIdService: ViewIdService,
+                          @Optional() protected _overflowService: OverflowService,
                           protected _logger: LoggerService) {
```

Comment:
Would not be a problem that the last dependency is not optional?

Reply:
Done